### ISSUE TITLE
[HOLD for payment 2023-09-06] [$500] Dev : Splash logo is grayed out

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#25617]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Launch the app

## Expected Result: 
Splash logo color is green

## Actual Result: 
Splash logo color is gray

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** dev
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

![staging](https://github.com/Expensify/App/assets/43996225/f7c99dee-c188-4dc5-a2a9-99b4ca69d2db)

![dev](https://github.com/Expensify/App/assets/43996225/d438e5ed-a361-44ac-a50e-8621a918e93d)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @situchan 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1691776351462309

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01118b202c69bb739e</li>
        <li>Upwork Job ID: 1694488036696375296</li>
        <li>Last Price Increase: 2023-08-24</li>
<li>Automatic offers: </li>
<ul>
<li>c3024 | Contributor | 26331939</li>
<li>situchan | Reporter | 26331942</li>
</ul></li>
    </ul>
</details>

**Comment by situchan** [comment]
@sakluger this is only reproducible on dev. Also confirmed by others - https://expensify.slack.com/archives/C049HHMV9SM/p1692173984147349

**Comment by sakluger** [comment]
I'm not sure if this is actually a bug - I asked in the Slack thread if the differently styled logo would ever end up on staging or prod. If not, I don't know if it's something we need to fix.

**Comment by sakluger** [comment]
Seems like this was the PR where the logo was changed: https://github.com/Expensify/App/pull/21551. Adding external label.

**Comment by studentofcoding** [comment]
# Proposal

## Please re-state the problem that we are trying to solve in this issue.
Dev logo is grayed out and not green like staging

## What is the root cause of that problem?
We are using `#002e22` instead of `#03D47C` on what we have on staging

## What changes do you think we should make in order to solve the problem?
We have to edit the .st2 of to be using `#03D47C` instead

https://github.com/Expensify/App/blob/37ac2522440a671ec5f75e4e0685a22597db35a0/assets/images/new-expensify-dev.svg#L2C3-L4

## What alternative solutions did you explore? (Optional)
None

## Result

https://github.com/Expensify/App/assets/20473526/00ef0e6f-b908-45e2-bd38-3a35451524cf

**Comment by studentofcoding** [comment]
# Proposal

## Please re-state the problem that we are trying to solve in this issue.
Dev logo is grayed out and not green like staging

## What is the root cause of that problem?
We are using `#002e22` instead of `#03D47C` on what we have on staging

## What changes do you think we should make in order to solve the problem?
We have to edit the .st2 of to be using `#03D47C` instead

https://github.com/Expensify/App/blob/37ac2522440a671ec5f75e4e0685a22597db35a0/assets/images/new-expensify-dev.svg#L2C3-L4

## What alternative solutions did you explore? (Optional)
None

## Result

https://github.com/Expensify/App/assets/20473526/00ef0e6f-b908-45e2-bd38-3a35451524cf


cc: @mananjadhav @sakluger @kavimuru

**Comment by sakluger** [comment]
Sorry for the late change here - after discussion with the team, this is a very small change and the bounty should be smaller.

**Comment by mananjadhav** [comment]
@c3024's proposal is good [here](https://github.com/Expensify/App/issues/25617#issuecomment-1687106784).

🎀 👀 🎀  C+ reviewed.

**Comment by situchan** [comment]
NOTE: https://github.com/Expensify/App/issues/25643 was closed in favor of this one, so I believe this should be fixed as well here.

**Comment by sakluger** [comment]
I went ahead and paid out the reporter bounty to @situchan

@mananjadhav could you please complete the BZ checklist before tomorrow? That way we can approve your manual payment right away after the 7-day hold.

**Comment by sakluger** [comment]
Paid @c3024 via Upwork.

We're just waiting on the BZ checklist to be completed now.

**Comment by mananjadhav** [comment]
https://github.com/Expensify/App/pull/21551 is the offending PR, and I've added a comment [here](https://github.com/Expensify/App/pull/21551#issuecomment-1710647510).

I don't think we need any changes in the process, it's a miss considering we also had 122 other file changes. Moreover considering this is Dev only and logo, I don't think we need a regression test either.

@sakluger I've raised my request on NewDot.
